docs/user_guide/HOWTO/IPA_Clients.rst

```diff
@@ -0,0 +1,86 @@
+.. _howto-ipa_clients:
+
+HOWTO Enroll Hosts into an IPA Domain
+=====================================
+
+Host should be able to join an IPA domain with a few catches. SIMP already uses
+the login stack that IPA uses, but it also actively manages the same resources
+that IPA provides automation for, like:
+
+#. autofs
+#. sudoers
+#. SELinux
+#. krb5
+
+These features may work in the future, but logins via SSSD or LDAP should work
+without issue.
+
+
+Adding clients
+--------------
+
+Adding clients requires two steps:
+
+#. Add the hosts on the IPA server, setting a one time password
+#. Running ``ipa-client-install`` on the host, using the password generated in
+   the previous step
+
+
+Add hosts to IPA
+^^^^^^^^^^^^^^^^
+
+There are two ways to complete this step:
+
+#. Use the IPA web interface, and write down the one time password
+#. Run ``ipa host-add`` on the command line and pregenerate the password
+
+Only option 2 will be covered here.
```

Comment:
If we're only going to do things one way, don't mention another. Might want to link people to the official RHEL docs though.